drake/examples/Cars/prius/README.md

```diff
@@ -0,0 +1,5 @@
+# Introduction
+
+This folder contains several models of the Prius sedan in both URDF and SDF format. The URDF and SDF files should be kimatically and dynamically equivalent with a tolerance smaller than the system's epsilon.
+
+There is an issue where the order of elements inside of the SDF matter. To verify this, prius_reordered.sdf includes XML elements that are in a different order relative to prius.sdf but are otherwise the same. Loading prius_reordered.sdf into Drake shows an obvious problem regarding the location of the front wheel.
```

Comment:
I will remove mentioning of this order-matters problem in this PR and address it separately in issue https://github.com/RobotLocomotion/drake/issues/1856. I will not create a failing unit test in this PR since https://github.com/RobotLocomotion/drake/issues/1846 is not yet resolved.